### PR TITLE
Add e2e tests for triage

### PR DESCRIPTION
**Pull request by eddieleeper-nb**
This is a work in progress and provides more test coverage for the call triage page

### CODE REVIEW

File: features/on_call_with_resident.feature [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  Scenario: Answer no to any children under 15
+    Given I am on a call with a resident
+    And I am editing the residents profile
+    When I choose "No" for any children under 15
+    And I save the edit resident form
+    Then I see a resident updated message
+    And the children under 15 details are "No"
+
+  Scenario: Answer yes to free prescriptions
```
Comment (rik86future) [inline features/on_call_with_resident.feature:46]: Maybe you could just use `Scenario Outline` here?

**Comment by eddieleeper-nb** [comment]
[Add e2e tests for triage](https://trello.com/c/lR3BZcqC/229-add-e2e-tests-for-triage)

**Comment by eddieleeper-nb** [comment]
I am aware of the `, allow_label_click: true` repetition on the radio button `choose` lines.
All works fine for me, using rack and firefox. (and fine in github)
If all good for the rest of the team I'll add it to the configuration, so that it's enabled by default.

**Comment by timwis** [comment]
Looks great! I fixed the merge conflicts that I caused in my other branch and will merge this once CI passes.

**Pull request merged**